### PR TITLE
Added envvars for e2e tests.

### PR DESCRIPTION
**Pull request by tmatsuo**
Fixing pubsub e2e test.

### CODE REVIEW

**Comment by tmatsuo** [comment]
Also running the full tests on our jenkins

**Comment by tmatsuo** [comment]
Seems like it needs to be `runtime: custom` for container engine e2e test.
Why it's deploying to App Engine flexible BTW?

**Comment by tmatsuo** [comment]
Now the e2e tests are passing on our jenkins :)

File: _tuts/6-pubsub.diff [hunk 1]
```diff
@@ -979,6 +979,89 @@ index c2e1dbb..104bb5d 100644
...
+   BOOKSHELF_CONFIG: /app/config/settings-e2e.yml
++  NGINX_CONF_INCLUDE: config/nginx-app.conf
++  PHP_INI_OVERRIDE: config/php-app.ini
+ 
+ runtime_config:
+   document_root: web
+diff --git a/test/e2e/BookshelfTest.php b/test/e2e/BookshelfTest.php
+deleted file mode 100644
+index 769e07d..0000000
+--- a/test/e2e/BookshelfTest.php
```
Comment (bshaffer) [inline _tuts/6-pubsub.diff]: Something is wrong with this - this file should not be affected by this change

Can you do a `git add -i` and not add this hunk? We don't want this to keep showing up in PRs.

**Comment by bshaffer** [comment]
LGTM

**Pull request merged**